docs/pages/guides/using-sentry.mdx

```diff
@@ -79,7 +79,8 @@ Configuring `@sentry/react-native` can be done through the config plugin. Add th
         "@sentry/react-native/expo",
         {
           "organization": "sentry org slug, or use the `SENTRY_ORG` environment variable",
-          "project": "sentry project name, or use the `SENTRY_PROJECT` environment variable"
+          "project": "sentry project name, or use the `SENTRY_PROJECT` environment variable",
+          "url": "https://sentry.io/ (or if you are using a self-hosted instance, the url of that e.g. https://self-hosted.example.com/)"
```

Comment:
```suggestion
          // If you are using a self-hosted instance, update the value of the url property
          // to point towards your self-hosted instance. For example, https://self-hosted.example.com/.
          "url": "https://sentry.io/"
```

Let's remove the explanation and instead use a comment. If the plugin example here is copy pasted into a project's app config, it might break because of that.